linux_sql/README.md

```diff
@@ -1,2 +1,24 @@
+
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+
+## Architecture and Design
+Here is an architectural diagram that shows a high level overview of this Project:
+![architecture diagram](assets/architecture_diagram.png)
+
+A psql instance that is stored on Server 1 is used to persist all the required data. 
+The bash agent on each server gather and insert that data into the psql instance. 
+Below is information on each file in the file directory (bash scripts and sql queries)
+
+### Script Descriptions
+- host_info.sh
+- This script only runs once at installation and is used to collect the host hardware information and then inserts it into the psql database.
```

Comment:
wrong indentation. this description should be a child item or the body of the bullet item.

Reply:
Fixed, thanks.